obspy/core/tests/test_event.py

```diff
@@ -773,12 +837,11 @@ def test_resources_in_global_dict_get_garbage_collected(self):
         # Now two keys should be in the global dict.
         rdict = ResourceIdentifier._ResourceIdentifier__resource_id_weak_dict
         self.assertEqual(len(list(rdict.keys())), 2)
-        # Deleting the objects should also remove the from the dictionary.
+        # Deleting the objects will no longer remove them from the dict, but
+        # the weak reference still behave as expected
```

Comment:
Can you please change this comment? It will be a bit confusing without having the history of the file at hand.